src/renderer/components/+welcome/__test__/welcome.test.tsx

```diff
@@ -51,23 +51,6 @@ describe("<Welcome/>", () => {
     WelcomeBannerRegistry.resetInstance();
   });
 
-  it("renders items in the top bar", async () => {
```

Comment:
Removing this since `<TobBar/>` won't live inside `<Welcome/>` component anymore. Same test case appears in topbar own test file.